helm-app-operator/pkg/helm/helm.go

```diff
@@ -0,0 +1,150 @@
+package helm
+
+import (
+	"context"
+	"fmt"
+
+	yaml "gopkg.in/yaml.v2"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+	"k8s.io/helm/pkg/chartutil"
+	"k8s.io/helm/pkg/engine"
+	"k8s.io/helm/pkg/kube"
+	cpb "k8s.io/helm/pkg/proto/hapi/chart"
+	"k8s.io/helm/pkg/proto/hapi/release"
+	"k8s.io/helm/pkg/proto/hapi/services"
+	"k8s.io/helm/pkg/storage"
+	"k8s.io/helm/pkg/tiller"
+	"k8s.io/helm/pkg/tiller/environment"
+	"k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
+
+	"github.com/operator-framework/helm-app-operator-kit/helm-app-operator/pkg/apis/app/v1alpha1"
+)
+
+const (
+	operatorName = "helm-app-operator"
+)
+
+// Controller can install and uninstall Helm releases given a custom resource
+// which provides runtime values for the Chart.
+type Controller interface {
```

Comment:
this is an odd name in the context of kubernetes where controller == has control loops.